medusa/providers/generic_provider.py

```diff
@@ -45,7 +46,6 @@
     InvalidShowException,
     NameParser,
 )
-from medusa.scene_exceptions import get_season_scene_exceptions
```

Comment:
If anyone is wondering why i've changed this..
You can't mock it in pytest when you import the method directly.